example/ios/example.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "group:example.xcodeproj">
+   </FileRef>
+   <FileRef
+      location = "group:Pods/Pods.xcodeproj">
+   </FileRef>
+</Workspace>
```

Comment:
I don't think we need any `xcworkspace` files. The xcworkspace is initially created when the user runs `pod install`. Is there a specific reason this file was added?

Reply:
This was part of the files changed when react-native was upgraded from an older version I used facebooks upgrade helper to do find these changes https://react-native-community.github.io/upgrade-helper/. I can revert these changes

Reply:
I think we can probably get rid of this file and add it to `.gitignore`. Really we should probably remove the `.xcworkspace` from the repo and ignore it too but it's ok to leave that for now.

Reply:
removed

Reply:
add .xcworkspace to gitignore